### PR TITLE
Remove some unnecessary allocations

### PR DESCRIPTION
**Pull request by AndrewWestberg**
# Description
Just some code cleanup for heap-allocated variables that are unused.

### CODE REVIEW

File: libs/non-integral/reference/non_integral.c [hunk 1]
```diff
@@ -241,7 +241,7 @@ int ref_exp(mpz_t rop, const mpz_t x)
 int findE(const mpz_t x)
 {
   mpz_t x_, x__, temp_q, temp_r;
```
Comment (teodanciu) [inline libs/non-integral/reference/non_integral.c]: These declarations of `temp_q` and `temp_r` are still here - is it on purpose?

Comment (AndrewWestberg) [reply libs/non-integral/reference/non_integral.c]: Will fix...

Comment (AndrewWestberg) [reply libs/non-integral/reference/non_integral.c]: resolved.

**Review by teodanciu — APPROVED** [review]
Looks good to me

**Pull request merged**